src/app/space/create/apps/apps.component.less

```diff
@@ -0,0 +1,22 @@
+@import (reference) '../../../../assets/stylesheets/shared/osio.less';
+
+.env-card-title {
+  text-transform: capitalize;
+}
+
+.resource-card {
+  background: lightgrey;
```

Comment:
update to use PatternFly color variables, as this will fail against Stylelint
`@color-pf-black-**`
(Adding the link to the color palette, since I'm not sure what shade of grey you are looking for: http://www.patternfly.org/styles/color-palette/)